### DISCUSSION TITLE
Texture file input for NASA lisf

### DISCUSSION THREAD

**Discussion by der-knight** [NASA-LIS/LISF#996] [Q&A] [answered]
Hi Lisf community
 I require a GLDAS soil texture file(as in https://ldas.gsfc.nasa.gov/gldas/soils) for generating the parameter processing file for NASAS LISF. The file location on website is https://ral.ucar.edu/research/land/technology/lsm/sfc_fields/STATSGO/topsoil30snew.gz but RAL seems to have moved it. 
I have the old 1gd4r file which i got from https://portal.nccs.nasa.gov/lisdata_pub/data/PARAMETERS/UMD/1KM/soiltexture_STATSGO-FAO.1gd4r.gz but i cannot process it. 
Is there any way i can get the high resolution stats go fao soil texture file.

Regards 
Nirdesh

**Comment by dmocko** [comment, marked as answer]
Hi Nirdesh,

It looks like the new location at RAL for these datasets is here:
    https://ral.ucar.edu/solutions/products/noah-multiparameterization-land-surface-model-noah-mp-lsm

For convenience for LIS users, I also made some of these datasets available at the LIS Data Portal:
    https://portal.nccs.nasa.gov/lisdata_pub/data/PARAMETERS/noah_2dparms/

David

**Reply by der-knight** [reply]
thank you David
